multidim-interop/go/v0.22/Makefile

```
@@ -0,0 +1,5 @@
+image.json: Dockerfile main.go go.mod go.sum
+	echo "{\"imageID\": \"$(shell docker build . -q)\"}" > $@
+
+clean:
```

```suggestion
.PHONY: clean
clean:
```

Nit pick, this is a `PHONY` target, right?

---

Correct, thanks